### DISCUSSION TITLE
Is it possible to open a local file with einkbro?

### DISCUSSION THREAD

**Discussion by fbrzvnrnd** [plateaukao/einkbro#435] [Q&A]
I tried with the file manager of Onyx Note3, but the "open with" function do not show me einkbro as app that could open a html file. Using a text editor app (Quoda) and asking to open the html with einkbro, I got a permission error opening the html... From einkbro I did not see a way to open local files... can you help me?

**Comment by fbrzvnrnd2** [comment]
Ok, my browser version was too old. The last version works very well:
